services/database/user.ts

```diff
@@ -35,4 +35,17 @@ async function getUsers() {
     return users;
 }
 
-export { getUser, getUsers };
+/**
+ * Creates a new user corresponding to newUserData and returns the newly created user
+ * @param newUserData - data corresponding to the new user
+ * @returns - the newly created user
+ */
+async function createUser(newUserData) {
```

Comment:
what's the type of `newUserData`? I would recommend defining a custom typed object for this.